src/main/java/core/basesyntax/filevalidatorservice/FileValidatorImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.filevalidatorservice;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class FileValidatorImpl implements FileValidator {
+    private LineValidatorImpl validator = new LineValidatorImpl();
+    private LineSpliterator lineSpliterator = new LineSpliterator();
+
+    @Override
+    public boolean validateFile(String path) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(path))) {
```

Comment:
it is better to separate fileReader and fileValidator into two different classes. At the moment you're trying to read the file and validate the data in a single method which is against Single responsibility principle.